github-management/kubernetes-repositories.md

```diff
@@ -220,6 +227,20 @@ associated projects. Many things seem vague. Help!**
 True, we need to improve these things. For now, do the best you can to conform
 to the spirit of the proposal (e.g. post the code of conduct, etc)
 
+**When I donate my project, am I transferring my copyrights?**
+
+No. All contributors retain ownership of their copyrights in the code they donate.
+Instead, they are granting a license to the project (that's the 'L' in 'CLA').
+
+For consistency and efficiency in complying with notice requirements, code that is
+donated to a Kubernetes repo should use [the standard header](https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.go.txt)
+referencing "The Kubernetes Authors". That doesn't mean you are transferring your
+copyright. Instead, it's a general reference to the fact that the copyrights remain
+owned by the authors of Kubernetes.
```

Comment:
I'm a little confused :see_no_evil: Just for my understanding:

> That doesn't mean you are transferring your copyright. Instead, it's a general reference to the fact that the copyrights remain owned by the authors of Kubernetes.

Does "remain owned" mean that the ownership of the copyright is transferred from the original authors/original company to the authors of Kubernetes?

Reply:
Hi @nikhita, no worries, if you are finding it confusing then I should reword it to make it clearer =)

The idea is that ownership for contributed code does not change. The owners of the copyrights might be individuals, or their employers, but in any case whoever owns the copyright before contribution still owns it afterwards. It’s just that a license is granted under the CLA. 

Let me stare at the language a bit more and try to clarify it better.

Reply:
I'm ok with merge and iterate-if-needed, if @dims and @justaugustus are good with this.